libcloud/compute/drivers/ec2.py

```diff
@@ -3630,7 +3630,7 @@ def ex_register_image(self, name, description=None, architecture=None,
                           image_location=None, root_device_name=None,
                           block_device_mapping=None, kernel_id=None,
                           ramdisk_id=None, virtualization_type=None,
-                          ena_support=None):
+                          ena_support=None, billing_product=None):
```

Comment:
Sure! That seems more accurate. I can make it plural. 
Since 'block_device_mapping' stuff was pre approved, we are not modifying it. right?
(Even though that should also be plural in this case)